test-packages/ts-template-imports-app/app/helpers/repeat.ts

```diff
@@ -0,0 +1,9 @@
+import { helper } from '@ember/component/helper';
+
+function repeat(params: [string, number] /*, hash*/): string {
+  return params[0].repeat(params[1]);
+}
+
+const repeatHelper = helper(repeat);
```

Comment:
why wrap in `helper`?

As of ember-source@4.5, we don't need helper, and can "just" export the functions (and change the signature to be a normal function as it once before it was moved in to this file)

Reply:
👍  Will fix.